examples/validation-app/src/interceptors/validate-phone-num.interceptor.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-validation-app
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  bind,
+  Interceptor,
+  InvocationContext,
+  InvocationResult,
+  Provider,
+  ValueOrPromise,
+} from '@loopback/context';
+import {CoffeeShop} from '../models';
+
+/**
+ * This class will be bound to the application as an `Interceptor` during
+ * `boot`
+ */
+@bind({tags: {key: ValidatePhoneNumInterceptor.BINDING_KEY}})
+export class ValidatePhoneNumInterceptor implements Provider<Interceptor> {
+  static readonly BINDING_KEY = `interceptors.${ValidatePhoneNumInterceptor.name}`;
```

Comment:
nitpick: I think usually we create a `keys.ts` file to expose the bingding keys. It's easy for organizing, especially when turn it into a component.

Reply:
it's generated by `lb4 interceptor`.